assets/js/script.js

```diff
@@ -127,6 +127,61 @@ function form() {
   var emailInputElement = document.querySelector('#email');
   var messageInputElement = document.querySelector('#message');
   var errorMessagesElement = document.querySelector('.error-messages');
+
+  class FormData {
```

Comment:
- [OPTIONAL] I think using a function instead of a class could keep this a bit simple

Reply:
@hiromataba  please check the source code because you sent comments over the compiled version and additionally we are using type module. We tested the prefilled on load an it's working.

Reply:
- On line 54 of your `index.html` Kindly Put the correct directory of your script 
![image](https://user-images.githubusercontent.com/75126481/141377095-4fa0f40e-aa4b-4761-8a3d-7ffc42eb53fe.png)
- It should be something like this `<script src="src/js/script.js" ></script>`

Reply:
Thank you for the comment, the src is `assets/js/script.js` because it's the compiled version of the script.